neurodamus/node.py

```diff
@@ -123,12 +122,9 @@ def register_node_manager(self, cell_manager):
 
     def _new_virtual_node_manager(self, circuit):
         """Instantiate a new virtual node manager explicitly."""
-        storage = libsonata.NodeStorage(circuit.CellLibraryFile)
-        pop_name = circuit.PopulationName
-        node_size = storage.open_population(pop_name).size
-        gid_vec = list(range(1, node_size + 1))
-        virtual_cell_manager = VirtualCellPopulation(pop_name, gid_vec)
-        self.virtual_node_managers[pop_name] = virtual_cell_manager
+        pop = libsonata.NodeStorage(circuit.CellLibraryFile).open_population(circuit.PopulationName)
```

Comment:
A nitpick:
how about keeping `pop_name = circuit.PopulationName` instead of of calling `circuit.PopulationName` every time ?

Reply:
to me it seems more clear, instead of having to look up what `pop_name` is

I think this should go away, anyways, so we handle all the connectivity the same; instead of creating VirtualPopulations out of thin air.